crates/turborepo-lockfiles/src/berry/mod.rs

```diff
@@ -0,0 +1,690 @@
+mod de;
```

Comment:
The PR title says yarn 3+, if this doesn't handle yarn 2 we should probably not call it 'berry' but yarn3 or something.